docs/tutorial/README.md

```diff
@@ -1,83 +1,138 @@
-up your first foundation, your repository will look like this:
+After completing these steps, your repository structure will resemble the following:
 
 ```shell
-$ tree
+`tree
 .
-└── foundations
-    └── my-foundation-dev
-        ├── README.md
-        └── platforms
-            └── gcp
-                └── README.md
+├── [README.md](http://readme.md/)
+├── compliance
+├── foundations
+│   └── likvid-foundation-dev
+│       ├── [README.md](http://readme.md/)
+│       └── platforms
+│           └── az
+│               └── [README.md](http://readme.md/)
+└── kit
+└── [README.md](http://readme.md/)`
+
+`7 directories, 4 files`
 ```
 
-Next, let's create our first kit module and apply it to our platform.
 
-```shell
-collie kit new "my-first-gcp-module"
-collie kit apply "my-first-gcp-module" --foundation my-foundation-dev --platform gcp
+### 2. Deploying Your First Kit
+
+First things first, for our journey, we need an organizational hierarchy for the Azure Cloud Platform. In this case, we will use the Collie Kit functionality to deploy it. To import the kit, use the following command:
+
+1. Run `collie kit import`.
+
+```
+Select a kit module from official hub modules ⌕
+**❯ Azure Organization Hierarchy azure/organization-hierarchy**
+ℹ 7/11 Next: ↓, Previous: ↑, Next Page: ⇟, →, Previous Page: ⇞, ←, Submit: ↵
 ```
 
-This generated a new executable terraform module at `kit/my-first-gcp-module` and a terragrunt module wrapper
-that simplifies executing that module (e.g. by managing common backend and provider configurations). at `foundations/my-foundation-dev/platforms/gcp`.
+2. Additionally, we need a `terragrunt.hcl` file for Terraform. Create it using `collie kit apply` and choose the `Azure Organization Hierarchy azure/organization-hierarchy` kit.
+
+3. When you run `cat foundations/likvid-foundation-dev/platforms/az/organization-hierarchy/terragrunt.hcl` on the file, you'll see that there are two additional files required to run our environment:
 
 ```shell
-22:07 $ tree
-.
-├── foundations
-│   └── my-foundation-dev
-│       ├── README.md
-│       └── platforms
-│           └── gcp
-│               ├── README.md
-│               └── terragrunt.hcl
-└── kit
-    └── my-first-gcp-module
-        ├── README.md
-        ├── documentation.tf
-        └── main.tf
+include "platform" {
+  path = find_in_parent_folders("platform.hcl")
+}
+
+include "module" {
+  path = find_in_parent_folders("module.hcl")
+}
 ```
 
-Inside the kit module, you can define any reusable set of functionality that your cloud foundation needs. A common
-module you will want to add to your kit is setting up organization-wide constraints at the root of the cloud
-resource hierarchy. Since a kit module is a standard terraform module, you can leverage official modules
-like `terraform-google-modules/org-policy/google` to set up organization policies.
+Copy this code block to `likvid-foundation-repo/foundations/likvid-foundation-dev/platforms/az/module.hcl`.
```

Comment:
nope - if we have to do that then the order of first deploying resource hierarchy and then bootstrapping won't work. 

The platfrom/module.hcl structure etc. comes with the bootstrap module templates

Reply:
of course, but than the bootstrap should run before or the user have to change the module and platform config for local running of tf